### PR TITLE
handful of updates for new site

### PR DESCRIPTION
**Pull request by patrickkettner**

### CODE REVIEW

**Comment by patrickkettner** [comment]
@ryanseddon could you give it a once over?

File: lib/build.js [hunk 1]
```diff
@@ -5,9 +5,22 @@
 var inBrowser = typeof define == 'function' && typeof define.amd == 'object';
 /* jshint -W117 */
 
+var _extend = function(a, b) {
```
Comment (ryanseddon) [inline lib/build.js]: Could we not just use lodash.extend? We're using it already in a few places.

Comment (patrickkettner) [inline lib/build.js]: its not loaded at this point in the browser context

Comment (ryanseddon) [inline lib/build.js]: Ah I see fair enough.

**Comment by ryanseddon** [comment]
LGTM :+1:

**Pull request merged**